src/Sylius/Component/Translation/spec/Fixtures/SampleTranslatableResource.php

```diff
@@ -11,11 +11,13 @@
 
 namespace spec\Sylius\Component\Translation\Fixtures;
 
-use Sylius\Component\Translation\Model\AbstractTranslatable;
+use Sylius\Component\Translation\Model\TranslatableInterface;
+use Sylius\Component\Translation\Model\TranslatableTrait;
 
 /**
  * @author Paweł Jędrzejewski <pawel@sylius.org>
  */
-class SampleTranslatableResource extends AbstractTranslatable
+class SampleTranslatableResource implements TranslatableInterface
```

Comment:
since you are using the 'TranslatableTrait' do you still have to implement the interface?